src/Rules/Violation.php

```diff
@@ -9,12 +9,12 @@ class Violation implements \JsonSerializable
     /** @var string */
     private $fqcn;
 
-    /** @var string */
-    private $error;
-
     /** @var int|null */
     private $line;
 
+    /** @var string */
+    private $error;
```

Comment:
this is the easiest way to get the sorting with `<=>` right. i think it makes more sense to have fqcn -> line number -> error.